formulas/github-get-archive/functions/github-get-archive

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+# Path to Github, Inc. repository.
+#
+#$1              - Path to Github repository.
```

Comment:
Single space.
Space after the `#`.
Actually, you could have copied and pasted my comment.  :-(